notebooks/Numba_053_profiling_the_compiler.ipynb

```diff
@@ -0,0 +1,1045 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Profiling the Compiler\n",
+    "\n",
+    "This notebook describes new features added in version 0.53 to help users measure time spent in the compiler."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from numba.core import config\n",
+    "# This is needed for recording LLVM pass timings used \n",
+    "# later in this notebook.\n",
+    "config.LLVM_PASS_TIMINGS = 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Tracking Compilation Events"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In Numba 0.53, a simple [event API](https://numba.readthedocs.io/en/0.53.0/developer/event_api.html) is implemented to allow users to track specific compilation events. This section will describe how it can be used to record compilation event within a region of code."
```

Comment:
```suggestion
    "In Numba 0.53, a simple [event API](https://numba.readthedocs.io/en/0.53.0/developer/event_api.html) is implemented to allow users to track specific compilation events. This section will describe how it can be used to record compilation events within a region of code."
```